app/src/main/java/uw/hcrlab/kubi/MainActivity.java

```diff
@@ -46,100 +39,34 @@ protected void onCreate(Bundle savedInstanceState) {
         //Notice: this is how each activity will get the robot and connect it to the robot face
         getWindow().addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
         setContentView(R.layout.activity_main);
-        robot = Robot.getInstance((RobotFace)findViewById(R.id.face), this);
-    }
-
-    /*
```

Comment:
yeah these seemed like clutter, I was wondering if we couldn't just delete them... :+1: